ci/build_wheel_cugraph.sh

```diff
@@ -5,8 +5,12 @@ set -euo pipefail
 
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 
+# Download the pylibcugraph wheel built in the previous step and make it
+# available for pip to find. We must use PIP_FIND_LINKS because the package
+# must be made available to the isolated build step, and there is no way to
+# manually install it into that environment.
 RAPIDS_PY_WHEEL_NAME=pylibcugraph_${RAPIDS_PY_CUDA_SUFFIX} rapids-download-wheels-from-s3 ./local-pylibcugraph
-python -m pip install --no-deps ./local-pylibcugraph/pylibcugraph*.whl
+export PIP_FIND_LINKS=`pwd`/local-pylibcugraph
```

Comment:
```suggestion
export PIP_FIND_LINKS=$(pwd)/local-pylibcugraph
```

I know we don't run `shellcheck` on our scripts, but if we did, it would complain about the use of backticks due to the rule below

- https://www.shellcheck.net/wiki/SC2006

Reply:
Good to know, thanks for the info!